ivy_models/googlenet/googlenet.py

```diff
@@ -0,0 +1,115 @@
+# global
+import ivy
+import ivy_models
+from .layers import ConvBlock, Inception, Auxiliary
+
+
+class GoogLeNet(ivy.Module):
+    """
+    Inception-V1 (GoogLeNet) architecture.
+
+    Args::
+        num_classes (int): Number of output classes. Defaults to 1000.
+        base_width (int): The base width of the ResNet. Defaults to 64.
+        v (ivy.Container): Unused parameter. Can be ignored.
+
+    """
+
+    def __init__(
+        self,
+        num_classes=1000,
```

Comment:
we aim to support dual format and spec or keyword arg initialisation, please also add them above, thanks!